### ISSUE TITLE
Abstract - typo around scope of WCAG 3 in first sentence

### ISSUE THREAD

**Issue by rickschmoo** [w3c/silver#242]
_"The W3C Accessibility Guidelines (WCAG) 3.0 provide a wide range of recommendations for making **web** content more accessible to users with disabilities"_

WCAG 2 = web
WCAG 3 = digital? Will submit a pull request.

**Comment by rickschmoo** [comment]
See PR https://github.com/w3c/silver/pull/243

**Comment by Myndex** [comment]
# OBJECTION
As I stated in survey, there are legal ramifications for the organization and for those developing guidelines or technology for guidelines. This is not a trivial matter.


"Digital" is exceedingly broad. It is important to point out that attempting to be that broad also means a potential increase legal liabilities. This adds a trickle down concern for those of us that are donating certain technologies, originally intended for web served content only.

As such I have to object — while WCAG is used for non-web development of apps anyway (because WCAG is a freely available standard) that's different than claiming in a statement that it's good for all things digital.

Do you want to create an implied liability for "digital" systems for clinical or medical use? Aerospace? Human safety?

Rethink this please. I am NOT saying it has to be limited to web, but that if you are going to extend, it also can not be at the wave of a hand, and the extensions need to be well defined.

Other standards agencies are very granular in the approach to what a given standard document covers.

**Comment by alastc** [comment]
Proposed response:

<hr> 

Thank you for the comment, and the PR (we do welcome PRs which speed up accepting things!).

However, WCAG 3 does have to claim a particular scope, which is web-technologies. That is slightly more expansive than web-content, but does not include other digital products. For example, WCAG is generally suited to products which have a user-agent (e.g. browser), but digital products that are stand-alone (e.g. kiosks) would require a lot of guidelines which WCAG does not provide.

Whilst WCAG can be applied more widely than web technologies, we cannot claim that within the standard due to the scope of the organisation and our working group.

We will look at re-wording other areas which may prevent this confusion in future.
